administrator/templates/atum/language/en-GB/en-GB.tpl_atum.ini

```diff
@@ -34,3 +34,4 @@ TPL_ATUM_TOGGLE_SIDEBAR="Toggle Menu"
 TPL_ATUM_TOOLBAR="Toolbar"
 TPL_ATUM_VIEW_SITE="View Site"
 TPL_ATUM_XML_DESCRIPTION="Continuing the Egyptian god/goddess theme (Khepri from 1.5, Hathor from 1.6, Isis from 3.0), Atum is the name of the Joomla 4 administrator template."
+TPL_BACK_TO_CONTROL_PANEL="Back to Dashboard"
```

Comment:
are you sure about putting this string back? Itss been removed from the main repo because it wasnt being used